examples/quad/main.rs

```diff
@@ -221,10 +221,22 @@ where
                 surface.supports_queue_family(family) && family.queue_type().supports_graphics()
             })
             .expect("No queue family supports presentation");
+
+        let device = &adapter.physical_device;
```

Comment:
nit: let's call this something different from `device`, which is typically reserved for logical devices